documentation/install.rst

```diff
@@ -138,6 +138,24 @@ For Ubuntu18+ run, an example for LLVM/Clang 10:
     sudo apt-get -y install  clang-10 llvm-10 libclang-10-dev llvm-10-dev
     sudo apt-get -y install  cmake make gcc g++
 
+For MacOSX:
+
+  Make sure the XCode is installed. If needed, install cmake, python and other utilities, e.g. using homebrew:
+  
+  .. code-block:: console
+    
+    brew install wget coreutils xz pybind11 cmake
+
+
+  Download and install the llvm+clang from the official site, e.g. using ``wget`` and 
+  add the location of llvm config to the $PATH:
+
+  .. code-block:: console
+    
+    wget --no-verbose https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.0/clang+llvm-11.0.0-x86_64-apple-darwin.tar.xz
+    tar -xJf clang+llvm-11.0.0-x86_64-apple-darwin.tar.xz
+    export PATH=$PATH:$(pwd)/clang+llvm-11.0.0-x86_64-apple-darwin/bin
```

Comment:
Why do we need this special instructions? We already have this script: https://github.com/RosettaCommons/binder/blob/master/build.py which take care of installing Pybind11, LLVM and compiling the later...

Reply:
This is a precompiled LLVM.   
Please note that it would be too "expensive" to compile the whole LLVM for each commit.
With the precompilled LLVM the whole pipeline takes ~ 5 min.

Reply:
aha, got it i see bin at the end of PATH now :)